pkg/coscheduling/coscheduling_test.go

```diff
@@ -1,622 +0,0 @@
-				st.MakePod().Name("pg2-1").UID("pg2-1").Namespace("ns1").Label(PodGroupName, "pg2").Obj(),
-			},
-			expected: framework.Success,
-		},
-		{
-			name: "pod belongs to podGroup1 and its PodGroupMinAvailable does not match the group's",
-			pod:  st.MakePod().Name("p").UID("p").Namespace("ns1").Label(PodGroupName, "pg1").Label(PodGroupMinAvailable, "2").Obj(),
-			pods: []*v1.Pod{
-				st.MakePod().Name("pg1-1").UID("pg1-1").Namespace("ns1").Label(PodGroupName, "pg1").Label(PodGroupMinAvailable, "3").Obj(),
-			},
-			expected: framework.Unschedulable,
-		},
-		{
-			name: "pod belongs to podGroup1 and its priority does not match the group's",
-			pod:  st.MakePod().Name("p").UID("p").Namespace("ns1").Priority(20).Label(PodGroupName, "pg1").Label(PodGroupMinAvailable, "2").Obj(),
-			pods: []*v1.Pod{
-				st.MakePod().Name("pg1-1").UID("pg1-1").Namespace("ns1").Priority(10).Label(PodGroupName, "pg1").Label(PodGroupMinAvailable, "2").Obj(),
-			},
-			expected: framework.Unschedulable,
-		},
-		{
-			name: "pod belongs to podGroup1, the number of total pods is less than minAvailable",
-			pod:  st.MakePod().Name("p").UID("p").Namespace("ns1").Label(PodGroupName, "pg1").Label(PodGroupMinAvailable, "3").Obj(),
-			pods: []*v1.Pod{
-				st.MakePod().Name("pg1-1").UID("pg1-1").Namespace("ns1").Label(PodGroupName, "pg1").Label(PodGroupMinAvailable, "3").Obj(),
-				st.MakePod().Name("pg2-1").UID("pg2-1").Namespace("ns1").Label(PodGroupName, "pg2").Label(PodGroupMinAvailable, "1").Obj(),
-			},
-			expected: framework.Unschedulable,
-		},
-		{
-			name: "pod belongs to podGroup2, the number of total pods is not less than minAvailable",
-			pod:  st.MakePod().Name("p").UID("p").Namespace("ns1").Label(PodGroupName, "pg2").Label(PodGroupMinAvailable, "3").Obj(),
-			pods: []*v1.Pod{
-				st.MakePod().Name("pg2-1").UID("pg2-1").Namespace("ns1").Label(PodGroupName, "pg2").Label(PodGroupMinAvailable, "3").Obj(),
-				st.MakePod().Name("pg1-1").UID("pg1-1").Namespace("ns1").Label(PodGroupName, "pg1").Label(PodGroupMinAvailable, "1").Obj(),
-				st.MakePod().Name("pg2-2").UID("pg2-2").Namespace("ns1").Label(PodGroupName, "pg2").Label(PodGroupMinAvailable, "3").Obj(),
-			},
-			expected: framework.Success,
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			cs := clientsetfake.NewSimpleClientset()
-			informerFactory := informers.NewSharedInformerFactory(cs, 0)
-			podInformer := informerFactory.Core().V1().Pods()
-			coscheduling := &Coscheduling{podLister: podInformer.Lister()}
-			for _, p := range tt.pods {
-				coscheduling.getOrCreatePodGroupInfo(p, time.Now())
-				podInformer.Informer().GetStore().Add(p)
-			}
-
-			podInformer.Informer().GetStore().Add(tt.pod)
-			if got := coscheduling.PreFilter(nil, nil, tt.pod); got.Code() != tt.expected {
-				t.Errorf("expected %v, got %v", tt.expected, got.Code())
-			}
-		})
-	}
-}
-
-func TestPermit(t *testing.T) {
```

Comment:
Although we have tests covering Permit() in core_test.go, but the logic in coscheduling.go (after getting result from core.go) are not covered. It'd be good to get those paths covered.

Reply:
Add

Reply:
I didn't see coscheduling_test.go